isso/views/comments.py

```diff
@@ -1141,6 +1142,12 @@ def login(self, env, req):
             isso_host_script = self.isso.conf.get("server", "public-endpoint") or local.host
             return render_template('login.html', isso_host_script=isso_host_script)
 
+    def logout(self, env, req):
```

Comment:
Since we're not doing anything to invalidate the cookies (or rather, the tokens), there's no reason to ping the server for anything. Just delete the cookie on the client side (or introduce a mechanism to invalidate the cookie on the server as well).